Dockerfile

```diff
@@ -0,0 +1,111 @@
+ARG BASE_IMAGE=ubuntu:20.04
+FROM ${BASE_IMAGE}
+
+# TODO: consider using iterative.ai or something else
+LABEL maintainer="dvc.org"
```

Comment:
shouldn't this be `dvcorg` since that's the user on DockerHub?

Reply:
It's what it had originally, though it might not be ideal. I usually see [email addresses](https://docs.docker.com/engine/reference/builder/#label).
https://github.com/iterative/cml/blob/89d16b31dd9d3a5d30416fcb1a973ea951317157/docker/Dockerfile#L7

Reply:
yes I'd say either support email address or user. `dvc.org` is weird (even `cml.dev` would be better).

Reply:
> _support email address_

Do we have that? 🤔

Reply:
```suggestion
LABEL maintainer="dvcorg <support@cml.dev>"
```

Reply:
(or if you prefer separating maintainer & email labels, I don't mind...)